### ISSUE TITLE
Should FOLDER.json be defined in the ehr.directory component?

### ISSUE THREAD

**Issue by ppazos** [openEHR/specifications-ITS-JSON#3]
Right now is under common: https://github.com/openEHR/specifications-ITS-JSON/blob/e141d638095292e8fca087d0496501df8c286365/components/RM/Release-1.0.4/Common/FOLDER.json

Ref to the class diagram: https://specifications.openehr.org/releases/RM/latest/UML/diagrams/RM-ehr.svg

**Comment by sebastian-iancu** [comment]
This repo file structure follows UML definition (xmi file) and specs in adoc:
https://specifications.openehr.org/releases/RM/latest/common.html#_folder_class

**Comment by ppazos** [comment]
@sebastian-iancu The link you sent shows the package definition common.directory, and the link I sent doesn't show the common package, I guess that one should be fixed.

Either way, the schema is on common/FOLDER, not on common/directory/FOLDER.

**Comment by sebastian-iancu** [comment]
Yes, you are right, the `directory` level is missing from the path - but this is not the only place, for several other subchapters I deliberately flattened same way so that a) structure is less complex b) xml-schema structure is in line with json-schema.
Is that level needed in a use-case that you have?
